lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -2096,6 +2104,10 @@ def main():
                 clone=module.params['clone'],
                 clone_permissions=module.params['clone_permissions'],
```

Comment:
It's too hacky, can we add wait param to this method and override the global wait?